crs-setup.conf.example

```diff
@@ -492,7 +492,7 @@ SecDefaultAction "phase:2,log,auditlog,pass"
 #    setvar:'tx.allowed_request_content_type=|application/x-www-form-urlencoded| |multipart/form-data| |multipart/related| |text/xml| |application/xml| |application/soap+xml| |application/json| |application/cloudevents+json| |application/cloudevents-batch+json|'"
 
 # Allowed HTTP versions.
-# Default: HTTP/1.0 HTTP/1.1 HTTP/2 HTTP/2.0
+# Default: HTTP/1.0 HTTP/1.1 HTTP/2 HTTP/2.0 HTTP/3 HTTP/3.0
 # Example for legacy clients: HTTP/0.9 HTTP/1.0 HTTP/1.1 HTTP/2 HTTP/2.0
```

Comment:
I don't fully get this line, should it also be updated with `HTTP/3 HTTP/3.0`?

Reply:
I meant line 496, the `Example for legacy clients`. But I think that I get it now, it is just be a custom change that you might do to include also `HTTP/0.9`, so it makes sense to include also all the defaults there. Updating it.

Reply:
Ah, I see, yes, that makes sense to update also.